src/databricks_ai_bridge/genie.py

```diff
@@ -133,7 +137,7 @@ def poll_result():
                     f"/api/2.0/genie/spaces/{self.space_id}/conversations/{conversation_id}/messages/{message_id}",
                     headers=self.headers,
                 )
-                if resp["status"] == "EXECUTING_QUERY" or resp["status"] == "COMPLETED":
+                if resp["status"] in ["EXECUTING_QUERY", "COMPLETED"]:
```

Comment:
nit: use a set instead of list
```suggestion
                if resp["status"] in {"EXECUTING_QUERY", "COMPLETED"}:
```